doc/api/packages.md

```diff
@@ -773,6 +757,20 @@ This section describes the fields used by the Node.js runtime. Other tools (such
 as [npm](https://docs.npmjs.com/creating-a-package-json-file)) may use
 additional fields which are ignored by Node.js and not documented here.
 
+The following fields in `package.json` files are relevant to the Node.js module
+loading system, and are described in detail below.
```

Comment:
Not just relevant, aren’t these the only fields that Node recognizes?

Reply:
Yes, I've simplified this line.